### PR TITLE
[ADP-2272] remove cli options related to blockfrost mode

### PR DESCRIPTION
**Pull request by paolino**
- [x] remove launch mode for blockfrost backend
- [x] remove launch specs for blockfrost backend
- [x] remove documentation related to blockfrost use 

### Issue Number

ADP-2272

### CODE REVIEW

**Comment by piotr-iohk** [comment]
Note that also docs require the update:
 - https://github.com/input-output-hk/cardano-wallet/blob/master/docs/user-guide/cli.md
 - https://github.com/input-output-hk/cardano-wallet/blob/master/docs/user-guide/common-use-cases/how-to-start-wallet-server.md

File: docs/user-guide/cli.md [hunk 1]
```diff
@@ -109,8 +109,7 @@ Serve API that listens for commands/actions. Before launching user should start
 :::{.cli}
 ```
 Usage: cardano-wallet serve [--listen-address HOST] 
-                            (--node-socket FILE [--sync-tolerance DURATION] | 
-                              --light --blockfrost-token-file FILE)
```
Comment (piotr-iohk) [inline docs/user-guide/cli.md:113]: `--light --blockfrost-token-file` also mentioned below in the cli.md doc I think.

Comment (paolino) [reply docs/user-guide/cli.md:113]: fixed

**Review by piotr-iohk — APPROVED** [review]
LGTM.

**Pull request merged**